docs/experiments/generic-pattern-matching.md

```diff
@@ -1,38 +1,77 @@
 ---
 append_help_link: true
-description: "Semgrep can match matches generic patterns in languages that it doesn’t yet support. You can use generic pattern matching for languages that don’t yet have a parser, configuration files, and other structured data, such as HTML or XML. Generic pattern matching is experimental."
+description: "Semgrep can match matches generic patterns in languages that it doesn’t yet support. You can use generic pattern matching for languages that don’t yet have a parser, configuration files, and other structured data such as XML. Generic pattern matching is experimental."
```

Comment:
```suggestion
description: "Semgrep can match generic patterns in languages that it doesn’t support yet. You can use generic pattern matching for languages that don’t have a parser, configuration files, or other structured data such as XML. Generic pattern matching is experimental."
```

"Semgrep can match matches generic patterns in languages that it doesn’t yet support." -> Looks like "matches" was a duplication?

Reply:
@s-santillan Just changed don't to do **not** - emphasising negative.